pkg/pulumi/setup.go

```
@@ -0,0 +1,25 @@
+	"github.com/pulumi/pulumi/sdk/v3/go/auto/optup"
+)
+
+func Setup(ctx context.Context) error {
+	st, err := createPulumiStack(func() (Stack, error) {
+		return &setupStack{}, nil
+	})
+	if err != nil {
+		return err
+	}
+	_, err = st.Up(ctx)
+	return err
```

golangci-lint (1.62.2)

error returned from interface method should be wrapped: sig: func (github.com/cardil/k1s/pkg/pulumi.Stack).Up(ctx context.Context, opts ...github.com/pulumi/pulumi/sdk/v3/go/auto/optup.Option) (github.com/pulumi/pulumi/sdk/v3/go/auto.UpResult, error)

(wrapcheck)